### PR TITLE
fix vendorStrip block on mobile

### PR DESCRIPTION
**Pull request by Arhell**

### CODE REVIEW

**Comment by Arhell** [comment]
Before

After

Now you can scroll the menu

**Comment by Arhell** [comment]
Issue #18759

**Comment by sftim** [comment]
Hi @Arhell 

I might not have the cycles to review all the PRs you want to assign my way. It's OK to submit PRs without assigning to a named individual.

**Comment by Arhell** [comment]
> Hi @Arhell
> 
> I might not have the cycles to review all the PRs you want to assign my way. It's OK to submit PRs without assigning to a named individual.

I understand, I'm sorry, but tell me who can still do a review on this topic?

**Comment by kbhawkey** [comment]
:eyes:

File: static/css/gridpage.css [hunk 1]
```diff
@@ -316,11 +316,6 @@ section.bullets .content {
 }
 
 @media screen and (max-width: 480px){
-    .gridPage #hero {
-        padding-right: 20px;
```
Comment (kbhawkey) [inline static/css/gridpage.css]: Hello @Arhell ,
Just looked at the changes.
Are the `.gridPage` changes essential?

File: assets/sass/_base.sass [hunk 2]
```diff
@@ -598,11 +601,11 @@ section
   #vendorStrip
     line-height: 44px
     max-width: 100%
-    overflow-x: auto
     -webkit-overflow-scrolling: touch
 
     ul
       float: none
+      overflow-x: auto
```
Comment (kbhawkey) [inline assets/sass/_base.sass]: Is there an advantage to move the `overflow-x: auto`?

File: static/css/gridpage.css (hunk 1 above)
Comment (Arhell) [reply static/css/gridpage.css]: I returned the changes, it seemed to me that it looks better

File: assets/sass/_base.sass (hunk 2 above)
Comment (Arhell) [reply assets/sass/_base.sass]: Yes, because if you leave it as it is, the whole block will move, but we need only the menu

**Comment by Arhell** [comment]
cc @kbhawkey

File: assets/sass/_base.sass (hunk 2 above)
Comment (kbhawkey) [reply assets/sass/_base.sass]: 👍

**Comment by kbhawkey** [comment]
@Arhell , I tested your changes by minimizing my browsers (Firefox and Chrome) to the smallest dimensions. I did not test on a mobile screen.
Viewing on a very small screen with Chrome, _without_ these changes, the list items were appropriately spaced.
/lgtm

**Comment by jimangel** [comment]
tested on firefox (pixel 4) and /lgtm
/approve

**Pull request merged**